src/HomePage.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import { useAuthState } from "react-firebase-hooks/auth";
+import { auth, db } from "./utils/firebaseConfig";
+
+import Result from "./pages/ResultsPage";
+import Game from "./components/Game";
+import WelcomePage from "./pages/WelcomePage";
+import UserHistory from "./components/UserHistory";
+import GameSkeleton from "./components/GameSkeleton";
+import LoginPage from "./pages/LoginPage";
+import { Button } from "./components/ui/button";
+
+import { useSharedState } from "@/context/sharedContext";
+
+function Home() {
+  const {
+    state: {
+      data,
+      anonymous,
+      results,
+      step,
+      start,
+      reset,
+      showResults,
+      isLoading,
+    },
+    dispatch,
+  } = useSharedState();
+
+  const navigate = useNavigate();
+
+  const [user, loading, error] = useAuthState(auth);
+
+  const notifyError = (message) => toast.error(message);
+
+  // useEffect(() => {
```

Comment:
Remove commented code.

This issue appears in almost every PR. Review your changes before submitting a pull request and review them once more afterwards.